### DISCUSSION TITLE
How should I properly remove the indicator for the Navigation Menu?

### DISCUSSION THREAD

**Discussion by james0r** [radix-ui/primitives#2688] [Help]
Using Next 14 and if I omit the Indicator component it breaks the animation in of my Content component. 

In other words,  when hovering over a trigger, the viewport animates in as expected but the content abruptly becomes visible at the end of the viewport animation. 

If I simply add the Indicator back into the Nav per the demo, the content animates in as expected. 

Any help much appreciated.

**Comment by benoitgrelard** [comment]
This sounds like a CSS issue rather than an issue with the primitives as the part is optional.
